httpie/output/writer.py

```diff
@@ -163,7 +164,10 @@ def get_stream_type_and_kwargs(
     if not is_stream and message_type is HTTPResponse:
         # If this is a response, then check the headers for determining
         # auto-streaming.
-        is_stream = headers.get('Content-Type') == 'text/event-stream'
+        ct_raw = headers.get('Content-Type', None)
+        if ct_raw:
+            ct, _ = parse_content_type_header(ct_raw)
+            is_stream = ct == 'text/event-stream'
```

Comment:
I find expliciter names much more suitable (with the rest of the code base).
```suggestion
        raw_content_type_header = headers.get('Content-Type', None)
        if raw_content_type_header:
            content_type_header, _ = parse_content_type_header(raw_content_type_header)
            is_stream = (content_type_header == 'text/event-stream')
 ```